core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -109,11 +109,11 @@ export class QuestionBackendApiService {
       skillId: string, offset: number,
       successCallback: (value: QuestionSummariesResponse) => void,
       errorCallback: (reason: string) => void): void|boolean {
-    const skillIds = [skillId];
+    const skillIds = skillId.split(',');
```

Comment:
Wait I'm bit confised by this, why do you split it? I thought that `skillId` is just one ID?

Reply:
`skillId` can be one id, but it is also possible that it is a string of multiple comma separated ids so I split it.

Reply:
From where is `_fetchQuestionSummaries` called with multiple skill IDs?

Reply:
I don't think it is ever called with multiple skill IDs. My thought was that the handler in the backend, `QuestionsListHandler`, supports receiving multiple skill IDs, so it should be possible to call `_fetchQuestionSummaries` with multiple skill IDs. And this is possible before this PR, so it should also be possible after this PR.

Reply:
No, this is confusing behaviour, if we wanted to pass multiple IDs to this function we would refactor it to accept list of IDs instead. Remove these changes.